file_store/src/file_info.rs

```diff
@@ -160,6 +160,8 @@ pub const LANDTYPE_DATA_SET: &str = "landtype";
 pub const SP_BOOSTED_REWARDS_BANNED_RADIO: &str = "service_provider_boosted_rewards_banned_radio";
 pub const VERIFIED_SP_BOOSTED_REWARDS_BANNED_RADIO: &str =
     "verified_service_provider_boosted_rewards_banned_radio";
+pub const VERIFIED_SUBSCRIBER_MAPPING_EVENT_REPORT: &str =
```

Comment:
Looks like you are missing a msg type.   There should be a definition for the report received by the ingestor ( which is your VERIFIED_SUBSCRIBER_MAPPING_EVENT_REPORT type ) and then there should be a definition for the report produced by the ingestor and consumed by the mobile verifier ( which is defined as VERIFIED_SUBSCRIBER_MAPPING_EVENT_INGEST_REPORT in my 846 pr )

Reply:
I think the name is wrong but one type is enough?

Reply:
Having looked at this again you are correct, we only need one type which is the VERIFIED_SUBSCRIBER_MAPPING_EVENT_INGEST_REPORT ( which is the wrapped mapping event report produced by the mobile ingestor and consumed by the mobile verifier )

In the past we have added a type for the report submitted to the mobile ingestor but as we never consume those reports over the info poller they are actually redundant and never used.  So the convention we have used in the past is actually incorrect in this instance.

Good with one type